daemons/controld/controld_te_events.c

```diff
@@ -455,6 +456,10 @@ process_graph_event(xmlNode *event, const char *event_node)
         }
     }
 
+    operation = crm_element_value(event, XML_LRM_ATTR_TASK);
+    if (!strcmp(operation, CRMD_ACTION_NOTIFY)) {
```

Comment:
Here, it would be better to either use safe_str_eq(), or check operation for NULL explicitly -- it helps make the controller resilient if a bug or memory corruption in one of the daemons causes the message to have no task.

Reply:
OK. I'll be careful.